src/pages/study-room/components/timer.tsx

```diff
@@ -1,15 +1,47 @@
 import { Button, Card, CardActions, CardContent, CardHeader } from '@mui/material';
+import { useMemo } from 'react';
+import useTimer from '../hook/useTimer';
+
+type KeyType = { [key in string]: string };
+const STEP_TEXT: KeyType = {
+  PLANNING: '계획',
+  STUDYING: '스터디',
+  RETROSPECT: '회고',
+  RESTING: '휴식',
+  COMPLETED: '완료',
+};
+
 const Timer = () => {
+  /** api호출
+   *
+   * const { isLoading, isError, data, error, refetch } = useQuery({
+   * queryKey: ['get-study-room-info'],
+   * queryFn: () => studyRoomInfo({ studyId: 1 }),
+   * });
+   *
+   */
+
+  const { seconds, startTimer, status } = useTimer({
+    updateAt: '2024-03-11T14:55:32.513Z',
+    step: 'PLANNING',
+  });
```

Comment:
format과 같은 option을 하나 더 만들어서 아예 useTimer 내부에서 화면에 보여줄 시간의 형식까지 맞춰주는것도 어떨까 싶네요~ 
```suggestion
  const { time, startTimer, status } = useTimer({
    updateAt: '2024-03-11T14:55:32.513Z',
    step: 'PLANNING',
    format: 'MM:SS'
  });
```